portal/index.php

```diff
@@ -51,8 +80,8 @@
     }
 
     if (count($result2) == 1) {
-        $defaultLangID = $result2[0]["lang_id"];
-        $defaultLangName = $result2[0]["lang_description"];
+        $defaultLangID = $result2[0]{"lang_id"};
+        $defaultLangName = $result2[0]{"lang_description"};
```

Comment:
These things keep coming back :)

Reply:
Blame it on PhpStorm for allowing me to have more than one project open at a time. Am I coping from 502, 503 or stable. Sorry, I did look for them though.